modules/docs/src/04-codegen/01-customisation.md

```diff
@@ -134,3 +134,26 @@ whenever the `adtMember` trait is in use.
 Note: The `adtMember` trait has NO impact on the serialization/deserialization behaviors of smithy4s.
 The only thing it changes is what the generated code looks like. This is accomplished by keeping the
 rendered schemas equivalent, even if the case class is rendered in a different place.
+
+#### Specialized collection types
+
+Smithy supports `list` and `set`, Smithy4s renders that to `List[A]` and `Set[A]` respectively. You can also use the `@uniqueItems` annotation on `list` which is equivalent to `set`.
+
+Smithy4s has support for two specialized collection type: `Vector` and `IndexedSeq`. The following examples shows how to use them:
+
+```kotlin
+use smithy4s.meta#indexedSeq
+use smithy4s.meta#vector
+
+@indexedSeq
+list SomeIndexSeq {
+  member: String
+}
+
+@vector
+list SomeVector {
+  member: String
+}
+```
+
+Both annotation are only available on `list`. You can't mix `@vector` with `@indexedSeq`, nor `@uniqueItems`.
```

Comment:
```suggestion
Both annotations are only available on `list`. You can't mix `@vector` with `@indexedSeq`, and neither one can be used with `@uniqueItems`.
```

Reply:
Addressed in [b3365de](https://github.com/disneystreaming/smithy4s/pull/303/commits/b3365de87c6af31653ec9810054c9d128c6a0e07)